bookmarklet.js

```diff
@@ -61,7 +61,7 @@ javascript:(
                                 if (errorMessage) {
                                     const errorType = (errorMessage.includes("Unexpected response status 400: The result is too big. Only queries with a maximum of 5000 results are supported.")) ? "ResultTooLargeError" : "InsightError";
                                     obj = {
-                                        title: "generated by https://github.com/ianmah/310-bookmarklet. Pls gib star",
+                                        title: `generated by https://github.com/ianmah/310-bookmarklet. Pls gib star${errorMessage}`,
```

Comment:
Nice suggestion. I think this might look a little cleaner. What do you think?
```suggestion
                                        title: `Generated by https://github.com/ianmah/310-bookmarklet. Pls gib star: ${errorMessage.trim()}`,
```

Reply:
cool! i don't know js well enough lmao